src/CLOptions.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-export interface ICLOptions {
+export interface CLOptions {
```

Comment:
Are there better names for `CLOptions` and `CLModelOptions`
It makes one sound like a superset of the other. Options should be all options and ModelOptions are those specific to models. 

However they are not related.  ModelOptions is independent and not a child of Options.

Seems like it would better be: `BotOptions` `ModelOptions`